src/rest-client/src/tokenHistory/api/TokenApi.js

```diff
@@ -338,57 +338,6 @@ class TokenApi {
             callback
         )
     }
-    /**
```

Comment:
Why did you remove this functions? If somebody is using this, they would get errors.

Reply:
I deleted it because no one used it. It was a function entered by an error in code.